client/components/dashboard/dashboard-model.js

```diff
@@ -133,6 +139,16 @@ explorer.components.dashboard.DashboardModel = function() {
    * @export
    */
   this.params = [];
+  
+  /**
+   * @type {{string}, {*}}
+   */
+  this.config = {
+    bigQuery: new BigqueryConfigModel()
+  };
+  
+  this.config.bigQuery.optimizeCurrentTimestamp.enabled = false;
```

Comment:
Do these settings need to be here? Shouldn't the "new BigqueryConfigModel" take care of this, and/or the v10 upgrader? If you have a BigqueryConfig abstraction, you should avoid having implementation details of its extensions leak through to general classes.